serf/Dockerfile

```
@@ -3,7 +3,7 @@
 # Stage1: build from source
 FROM ghcr.io/cybozu/golang:1.23-noble AS build
```

Could you update base image too?
```suggestion
FROM ghcr.io/cybozu/golang:1.24-noble AS build
```

---

update